sec_certs/rules.yaml

```diff
@@ -0,0 +1,1178 @@
+---
```

Comment:
Seems ok. Please note that we must add this file into `package_data` in `setup.py`

Reply:
Done. However, the way the other files are present in `package_data`, do they actually get included due to that? They are in subpackages and not in the root and the strings in the`package_data` do not have the subpackages.

Reply:
I created a wheel (after you merged) with `python3 -m build .` and inspected its contents. All files specified in `package_data` are present in the wheel. So we should be safe.